dev/ci/c-bedrock2.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+bedrock2_CI_DIR="${CI_BUILD_DIR}/bedrock2-ci"
```

Comment:
Why not point to the bedrock2 repo instead? This will make it more difficult for us to submit patches upstream.

Reply:
I see, so you're saying that if a Coq change breaks bedrock2 and you create a patch, you'd like to make only one PR (against bedrock2) and avoid an extra PR against bedrock2-ci, right?

bedrock2 depends on two other repos (bbv and riscv-coq), and they have to be separate repos. We did spend some thought on how to put everything together for CI, see eg [here](https://github.com/mit-plv/bedrock2/issues/12#issuecomment-399248610). The main concerns are diamond dependencies and having a fast workflow.

So, an alternative CI setup which would be more convenient for the Coq developers would be to have bbv and riscv-coq as git submodules of bedrock2. Like this, if a patch for bbv or riscv-coq is needed, it would still require two PRs, but patches for bedrock2 (where most of the code lives) would only require one PR. This seems possible, but once we get bitten by diamond dependencies on bbv, we'll have to complicate the build setup of bedrock2 a bit by allowing it to depend on a bbv at a different path, ignoring its bbv submodule. I'll keep thinking about how to do that nicely.

@maximedenes  does that make sense, or did I misunderstand something?